frontend/__tests__/state-functions.test.js

```diff
@@ -0,0 +1,5 @@
+describe('test', () => {
```

Comment:
small things, just JS-world styles :)
1. test folder is just `test`, no need in underscores. What the meaning they bring?
2. `.test.` usually is called `.spec.` 
3. generally tests are placed in the same directory where the component is, so you have a folder `SomeComponent` and there
SomeComponent
SomeComponent

Reply:
Sure. It was in some article and I just followed it. Will be fixed in next push.